test/StreamTest.php

```diff
@@ -290,13 +304,14 @@ public function testReadReturnsEmptyStringWhenAtEndOfFile()
         $this->assertEquals('', $stream->read(4096));
     }
 
-    public function testGetContentsReturnsEmptyStringIfStreamIsNotReadable()
+    public function testGetContentsRisesExceptionIfStreamIsNotReadable()
     {
+        $this->setExpectedException('RuntimeException');
         $this->tmpnam = tempnam(sys_get_temp_dir(), 'diac');
         file_put_contents($this->tmpnam, 'FOO BAR');
         $resource = fopen($this->tmpnam, 'w');
         $stream = new Stream($resource);
-        $this->assertEquals('', $stream->getContents());
+        $stream->getContents();
```

Comment:
Yes and no. I can see an argument that the current behavior is broken, as it leads to unexpected results.

BTW, @mtymek - move the `setExpectedException()` call to immediately before the operation that will raise the exception; otherwise, an exception happening earlier will lead to a false positive result.